<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSegmentationWidget</class>
 <widget class="QWidget" name="ImageSegmentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageSegmentationWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <row>
        <property name="text">
         <string>Seed Point</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>X</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Y</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="intensityProbBtn">
       <property name="text">
        <string>Load Intensity Probs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="textureProbBtn">
       <property name="text">
        <string> LoadTexture Probs</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="probabilityBtn">
         <property name="text">
          <string>Compute Probability</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="segmentBtn">
         <property name="text">
          <string>Segment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newSeedBtn">
         <property name="text">
          <string>Choose New Seed</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>intensityProbBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageSegmentationWidget</receiver>
   <slot>loadIntensityProbs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textureProbBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageSegmentationWidget</receiver>
   <slot>loadTextureProbs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>probabilityBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageSegmentationWidget</receiver>
   <slot>computeProbability()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>segmentBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageSegmentationWidget</receiver>
   <slot>segment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageSegmentationWidget</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newSeedBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageSegmentationWidget</receiver>
   <slot>newSeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadIntensityProbs()</slot>
  <slot>loadTextureProbs()</slot>
  <slot>computeProbability()</slot>
  <slot>segment()</slot>
  <slot>save()</slot>
  <slot>newSeed()</slot>
 </slots>
</ui>
